addons/gamification/models/challenge.py

```diff
@@ -543,42 +538,49 @@ def report_progress(self, users=(), subset_goals=False):
 
         MailTemplates = self.env['mail.template']
         if challenge.visibility_mode == 'ranking':
+            template_id = self.env.ref('gamification.simple_report_template_ranking', raise_if_not_found=False)
```

Comment:
👎 don't hardcode reference to a template, it makes it unremovable or painful to be configured by external modules